### PR TITLE
My list groups refactor

### PR DESCRIPTION
**Pull request by gyurka00**
New package in the project!
Do not forget the `yarn install`!

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit 53d9b319769f0a970c8a0f442d6625c09fbd4ba7

https://app.netlify.com/sites/augmint/deploys/5ad9e4dfb13fb11110adcd71

File: src/components/augmint-ui/header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import { StyledHeaderH3 } from "./styles";
```
Comment (reyrodrigez) [inline src/components/augmint-ui/header/index.jsx]: this file should handle other header sizes as well (h1, h2, ...)

File: src/components/augmint-ui/header/styles.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+import theme from "styles/theme";
+
+const BaseHeader = `
+    color: ${theme.colors.white};
+`;
+
+export const StyledHeaderH3 = styled.h3`
+    ${BaseHeader};
```
Comment (reyrodrigez) [inline src/components/augmint-ui/header/styles.js]: where are all the other styles for h3 coming from? if the appearance of this element is still depending on external stylesheet we should port those styles to the scoped style.js.

File: src/components/augmint-ui/header/styles.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import theme from "styles/theme";
+
+const BaseHeader = `
+    color: ${theme.colors.white};
+    font-family: MaisonNeue,'Helvetica Neue',Arial,Helvetica,sans-serif;
```
Comment (reyrodrigez) [inline src/components/augmint-ui/header/styles.js]: how about `${theme.typography.fontFamilies.default};`?

File: src/components/augmint-ui/header/styles.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    font-weight: 200;
+    padding: 0;
+`;
+
+const HeaderH3 = `
+    font-size: 26px;
```
Comment (reyrodrigez) [inline src/components/augmint-ui/header/styles.js]: ... and `${theme.typography.fontSizes.h3};` also please convert these values to rem: `24/16 = 1.5rem`

File: src/components/augmint-ui/header/styles.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    color: ${theme.colors.white};
+    font-family: ${theme.typography.fontFamilies.default};
+    font-weight: 200;
+    padding: 0;
+`;
+
+const HeaderH3 = `
+    font-size: ${theme.typography.fontSizes.h3};
+    line-height: ${theme.typography.fontSizes.h3};
+    margin-bottom: 0,875rem;
```
Comment (reyrodrigez) [inline src/components/augmint-ui/header/styles.js]: `,` --> `.`

**Pull request merged**